src/services/navigation/useNavLinks/useStaticNavLinks.js

```diff
@@ -409,6 +409,11 @@ export function useStaticNavLinks() {
     login: {
       text: 'Login',
       path: () => {
+        // Enterprise login page is at different url than Cloud; see App.tsx
+        if (config.IS_SELF_HOSTED) {
```

Comment:
Do we still need these conditionals with this change? https://github.com/codecov/gazebo/blob/b07fcc7bae68f7a015dac9804f8649a97b6b5859/src/App.tsx#L72-L79

Reply:
Ah thanks, yeah, good question! So I thought I'd leave that there because-

1. If we're confident the only way a user would end up at the Login Page is via a link/redirect from our app (via `useStaticNavLinks`), we could remove the logic in App.tsx of which component renders when url is `https://{host}/login`. But I thought if say the Enterprise user had `/login` bookmarked, I wouldn't want to show the incorrect page of the Cloud login page to them after this PR.. Or say if another service had a callback url registered somewhere I don't know about, wouldn't wanna break that..

2. I thought it'd be nice if the routing logic that's both in App.tsx and `useStaticNavLinks` would remain consistent. So that would now include this instance of the "Login Page" being at `/` for Enterprise and `/login` for Cloud.

Reply:
Yeah thats a fair point! Having both "shores up" any possibility of a user trying to go to either `/` or `/login` when they should be routed to the other. Down to keep it